### ISSUE TITLE
Consider hosting in separate org.

### ISSUE THREAD

**Issue by ngokevin** [aframevr/cli#1]
Can we host this on the aframevr-userland org in the meantime? We can add it to the Getting Started page and promote on Slack so people can try it out. Next time, we would appreciate engaging with the community and discussing before taking action.

In #planning, we discuss about similar things like what we should promote to the org such as useful components. Our concern is that adding projects to the org is a liability because people will expect a minimum level of quality and support. We only want to host projects that we feel comfortable to maintain long term. 

Personally, I'm not entirely sold that a Node CLI should be an official promoted way to get started with A-Frame yet. If we're targeting beginners, Node and npm are difficult to set up. Creating Glitches and listing them near the each doc page's table of contents is something I've been leaning more to.

Thanks for understanding.

**Comment by cvan** [comment]
[`ionic`](https://ionicframework.com/docs/cli/) has a CLI that has been serving them quite well. The [`aframe` CLI](https://github.com/aframevr-userland/aframe-cli) is a simple scaffolder, local dev server (with live-reloading, backed by `live-server`), and can deploy projects to a static CDN.

This tool doesn't require knowledge of Node, npm, nor git. A user must only first `brew install node`; and `npm` is available. Then it's just a matter of running `npm install -g aframe-cli`. And the user now has the `aframe` program to access from the command line.

Following up on this will be a native menu-bar app and WebExtension browser extension that will side-load install and run the `aframe` command behind the scenes.

I'll maintain myself as the owner, fielding issues, pull-requests, and providing a quality level of support. These tools will mature to satisfy the needs of web developers and designers who want to develop A-Frame in a low-friction way without needing to be connected to the Internet (e.g., Glitch).

For any issues you encounter or feature requests you have for the current `aframe` CLI tool, you can be file them here, and I'll be on top of them: https://github.com/aframevr-userland/aframe-cli/issues

**Comment by ngokevin** [comment]
I disagree. I don't understand how we claim this tool doesn't require knowledge of Node or npm, but then the next step is install Node and then run an npm command and then a dev server and a deploy step?  I've given workshops where you had to run a live server, people got stuck getting Node, npm, running the command, not understanding ports. You also have to install brew, and that's just on OS X. On Windows, it becomes much tougher. That is friction.

Glitch would absolutely be easier. You click a link, and you get started immediately. No installation, no local server, no need to choose a text editor, no deployment. Needing an internet connection is not an issue, developers have them, they can export code from Glitch if they plan on being off line, and you need an internet connection to install the CLI anyways.

This is just from my experience from haven interfaced with people and helping them get started, I'd default to not making this official until proven otherwise. I don't think this is the way to go. I encourage these ideas to be proven first rather than speculated or forced upon.

**Comment by ngokevin** [comment]
I've also built Ionic apps before. Their tools are much more complex because they need to get native code to wrap web apps to run on mobile. A-Frame can function with just a HTML file. Different projects have different needs.

**Comment by ngokevin** [comment]
Just to give a decent heads up, I plan on removing this repo in the next couple of days. I do want to call out we should not create repositories on the official org without telling anyone on the team. If the project evolves or gains traction, we can discuss and consider it. Nothing against the project itself, I'd be happy if it evolved and filled some gaps, but these things should happen properly rather than pulling the trigger without mention. If necessary, we can evolve the code of conduct document to draw the lines.
